CONTRIBUTING.md

```diff
@@ -48,6 +48,13 @@ Every additional section will slow down the prompt a little bit. If your section
 * **Good:** check if command exists, check the value of environment variable
 * **Bad:** network requests, reading large files, etc
 
+### What is the naming convention for options?
+
+All options of prompt follow a specific pattern so that it is easy to remember -> `SPACESHIP_${OPTION}_${PROPERTY}`.
```

Comment:
I'd say `SPACESHIP_SECTION_OPTION[_ROPERTY]` or something like that.

@salmanulfarzy your thoughts?

Reply:
`${}` makes them look like to be configured on runtime. I think `SPACESHIP_SECTION_OPTION[PROPERTY]` sounds good. 

Specifying both `OPTION` and `PROPERTY` may cause confusion. It might be better if only one of them is specified.

Reply:
I meant cases when we make an option like `SPACESHIP_GIT_BRANCH_COLOR` where:

* `GIT` is a section
* `BRANCH` is an option or _namespace_
* `COLOR` is a property

Or `SPACESHIP_KUBECONTEXT_NAMESPACE_SHOW `, or other ones. 

The main goal is to make clear guidelines on how to name new options. Whether it should be a `SPACESHIP_KUBECONTEXT_NAMESPACE_SHOW` or  `SPACESHIP_KUBECONTEXT_SHOW_NAMESPACE`.

Reply:
Distinction between option and property is might be blurry. So better to add some relevant examples for that @Ashniu123

Reply:
Looks like there isn't one pattern that can describe all options, maybe the best thing to do is to describe a reasoning you have in mind and give a couple of examples.

For example, I can see now we have:

```
SPACESHIP_USER_COLOR
SPACESHIP_USER_COLOR_ROOT
SPACESHIP_DIR_LOCK_COLOR
SPACESHIP_GIT_STATUS_DIVERGED_COLOR
```

I'd simply say: 

When naming new properties, keep unique parts of the name to the end.

Good:

```
SPACESHIP_GIT_STATUS_COLOR_BEHIND
SPACESHIP_GIT_STATUS_COLOR_DIVERGED
```

Bad:

```
SPACESHIP_GIT_STATUS_BEHIND_COLOR
SPACESHIP_GIT_STATUS_DIVERGED_COLOR
```

Reply:
#594